CommandHandlerTesterLib/TestLongActionPlugin.cs

```diff
@@ -0,0 +1,52 @@
+﻿using CA_DataUploaderLib;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace CommandHandlerTesterLib
+{
+    public class TestLongActionPlugin : LoopControlPlugin
+    {
+        public TestLongActionPlugin(CommandHandler cmd) : base(cmd)
+        {
+            AddCommand(nameof(LongAction), LongAction);
+            AddCommand("help", HelpMenu);
+        }
+
+        private bool HelpMenu(List<string> _)
```

Comment:
could do in another PR, but plan to change it so that if I implement a plugin command, I always have help. Potentially the base class would be LoopControlCommand with a command name and description. It could also help take out the boilerplate around the action, as we could take some of the boilerplate away from the action, so that we get login automatically and potentially a separate action that runs on timeouts (instead of explicitely having to catch the TaskCanceledException).

Reply:
Please do.

Reply:
Implemented in latest PR